src/build/cargo.rs

```diff
@@ -85,9 +85,17 @@ pub(super) fn cargo(internals: &Internals, package_arg: PackageArg, progress_sen
         }
         Ok(cwd) => BuildResult::Success(cwd, vec![], vec![], true),
         Err(err) => {
+            // This message goes like this to the UI via showMessage. In VSCode
+            // this ends up on one single line, so it's important to keep it concise.
             let stdout = String::from_utf8(out_clone.lock().unwrap().to_owned()).unwrap();
-            debug!("cargo failed\ncause: {}\nstdout: {}", err, stdout);
-            BuildResult::Err(err.to_string(), None)
+            let stdout_msg = if stdout.is_empty() {
+                "".to_string()
+            } else {
+                format!("({})", stdout)
+            };
+            let msg = format!("Cargo failed: {}{}", err, stdout_msg);
+            info!("{}", msg);
+            BuildResult::Err(msg, None)
```

Comment:
This message is longer than previously, so I don't think it makes it more concise.

Reply:
Really? I mean I tested this live (see screen grab at the top).

The current code would show:

```
cargo failed
cause: Could not compile `getopts`.
stdout:
```

Which has two problems:

1. VSCode only shows one line. So the user would only see `cargo failed`, not the cause.
2. It prints `stdout:` regardless of there being anything to show there.

The proposed change would do 

```
Cargo failed: Could not compile `getopts`.
```

And if there is a stdout pad that on inside parentheses. I maintain this is a better solution.